plugins/woocommerce/includes/tracks/events/class-wc-settings-tracking.php

```diff
@@ -230,14 +230,9 @@ public function track_settings_page_view() {
 	 * @param string $hook Page hook.
 	 */
 	public function possibly_add_settings_tracking_scripts( $hook ) {
-		// phpcs:disable WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.NonceVerification
-		if (
-			! isset( $_GET['page'] ) ||
-			'wc-settings' !== wp_unslash( $_GET['page'] )
-		) {
+		if ( ! is_wc_admin_settings_page() ) {
```

Comment:
Here we are losing `wp_unslash( )`. Can we asses the impact please and possibly verify why it was added here?

Reply:
I thought it was simpler to just add it to the `is_wc_admin_settings_page` function instead (https://github.com/woocommerce/woocommerce/pull/59069/commits/956c99d835611b803dc7cb88b05ac7ca0ddca6dd)